cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISink.java

```diff
@@ -871,6 +874,37 @@ private void accumulateByEntityType(NGSIEvent event) {
             batch.addEvent(destination, event);
         } // accumulateByEntityType
 
+        private void accumulateByEntityId(NGSIEvent event) {
+        	Map<String, String> headers = event.getHeaders();
+            ContextElement originalCE = event.getOriginalCE();
+            ContextElement mappedCE = event.getMappedCE();
```

Comment:
Missidentent block:

![imagen](https://user-images.githubusercontent.com/1534240/78289687-d882a580-7522-11ea-8c48-dcb89cc63ea0.png)

Maybe some "spaces vs. tabs" issue?

Reply:
(In general, I think the code should be reviewed for indent problems... not sure if the ones I have commented in this PR are the only ones. They are easy to find with a carfeful examination of https://github.com/telefonicaid/fiware-cygnus/pull/1850/files).

Reply:
Fixed in [72fd80e](https://github.com/telefonicaid/fiware-cygnus/pull/1850/commits/72fd80eef59c95ee8ade44adb2e4751144649009)